app/src/main/java/com/example/musicapp/dto/Instrument.kt

```diff
@@ -1,6 +1,13 @@
 package com.example.musicapp.dto
 
-data class Instrument(var id: Int, var name: String, var type: String) {
+import com.google.gson.annotations.SerializedName
+
+data class Instrument(
+    @SerializedName("id")var id: Int,
+    @SerializedName("name") var name: String,
+    @SerializedName("name") var type: String
```

Comment:
Configured to @SerializedName annotation to serialize a field with a different name instead of an actual field name: https://www.tutorialspoint.com/what-to-use-serializedname-annotation-using-gson-in-java#:~:text=The%20%40SerializedName%20annotation%20can%20be,of%20an%20actual%20field%20name.

Reply:
Since they are the same, in this case, @SerializedName isn't adding any value here... at the moment.  But, it will if the names change on either side (JSON or attribute names), and the spacing on each line is more readable.